tasks/dashboards.yml

```diff
@@ -123,3 +137,58 @@
 #   no_log: true
 #   with_fileglob:
 #     - "/tmp/dashboards/*"
+
+- name: Create/Update dashboards file (provisioning)
+  become: true
+  copy:
+    dest: "/etc/grafana/provisioning/dashboards/ansible.yml"
+    content: |
+      apiVersion: 1
+      providers:
+       - name: 'default'
+         orgId: 1
+         folder: ''
+         type: file
+         options:
+           path: /var/lib/grafana/dashboards
+    backup: false
+    owner: root
+    group: grafana
+    mode: 0640
+  notify: restart grafana
+  when: grafana_use_provisioning
+
+- name: Install rsync
+  package:
+    name: ["rsync"]
+  when: grafana_use_provisioning
+
+- name: Import grafana dashboards through provisioning
+  synchronize:
```

Comment:
Why `synchronize` and not `copy`?

Reply:
I use synchronize because this module can synchronize the content of the provisioning dashboard directory. If I remove one and add another dashboard, after the ansible provisioning, the destination directory will be in sync correctly, if the property grafana_provisioning_synced is true. With copy I need to look the content of the destination dir, compare it to the source dir, etc ...

Reply:
I don't like it because with synchronize we need to have external handling of file permissions but I accept your arguments and let's do it this way.

Before we merge it please move rsync installation to `prepare.yml` playbook in `molecule/default/prepare.yml` as this has nothing to do with a role and will just slow down execution. Also add a note in README.md that rsync is required on deployer and target machines.

Reply:
Me too I'd rather prefered not to use synchronize, but it's kindy easier.